catalog/ui/catalog-ui-search/src/main/webapp/component/user-settings/user-settings.hbs

```diff
@@ -26,7 +27,7 @@
     </button>
     <button class="navigation-choice is-neutral choice-search">
         <span class="fa fa-search"></span>
-        <span>Query</span>
+        <span>Search Settings</span>
```

Comment:
Adding Settings seems redundant to me.

Reply:
“Search Options” should be sufficient and less redundant, and minimize any appearance of looking like a search area due to magnifying glass icon.

Reply:
fixed